main.go

```diff
@@ -45,3 +53,20 @@ func main() {
 		log.Fatalf("Failed to submit list of installed packages; err = %v", err)
 	}
 }
+
+func local() {
```

Comment:
would be nicer if `error` is returned and handled outside of the function

you should also return the payload, not printing it inside